lib/routes/receive_payment/lightning/widgets/compact_qr_image.dart

```diff
@@ -49,7 +49,7 @@ class CompactQRImage extends StatelessWidget {
   final double? size;
   final bool bip21;
 
-  const CompactQRImage({super.key, required this.data, this.size, this.bip21 = false});
+  const CompactQRImage({super.key, required this.data, this.size, this.bip21 = true});
```

Comment:
Assuming this is a bolt11 invoice (lightning widget), should bip21 be set to `false`?